packages/shared/subpaths.mjs

```diff
@@ -1,5 +1,5 @@
 // This file is a helper for the "subpath-workaround.mjs" script
-// We have to polyfill our "exports" subpaths :cry:
+// When adding an entry to "subpathNames" also add it to "files" in package.json
 
 export const subpathNames = [
```

Comment:
Can't we just replace this with package.files ? In order to use package.json as the source of truth ?

Reply:
The `files` array contains directories (like `dist`, `scripts`, `react`) that need special handling and/or shouldn't get subdirectories created. So one way or another you'd need to add special handling. I can adjust the logic in another PR probably so that `files` array is the source of truth